keyboards/keychron/c3_pro/config.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2023 @ Keychron(https://www.keychron.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Indication led */
+#define LED_CAPS_LOCK_PIN A3
+#define LED_MAC_OS_PIN A4
+#define LED_WIN_OS_PIN A5
+#define LED_PIN_ON_STATE 1
```

Comment:
1 is the default for the on state, so no need to define it here.  And capslock should be moved to info.json. 
```suggestion
/* Indication led */
#define LED_MAC_OS_PIN A4
#define LED_WIN_OS_PIN A5
```

Reply:
Hi, There is a compilation error when i deleted the LED_PIN_ON_STATE from config.h,.
![image](https://github.com/qmk/qmk_firmware/assets/66767061/482d03b8-0da7-4d44-9864-cd41e82a4a92)